### ISSUE TITLE
Ultra Wide Screen Unplayable

### ISSUE THREAD

**Issue by Zaithe** [jasonrohrer/OneLife#100]
When trying to play the game on my 3860x1080 screen the hitboxes for everything is offseted to the left by about 500px or so. Tried getting out of the tutorial but its having so much trouble i just end up closing it. Also everything is letterboxed. Looks like this has zero ultra wide screen support.

**Comment by jasonrohrer** [comment]
Can you go into Settings and try Borderless fullscreen mode?  I know it will be hard to click, but that should fix it.

**Comment by jasonrohrer** [comment]
I'm still waiting for confirmation that this works.  If it does, then I will make borderless the default behavior on ultrawidescreen.

**Comment by oddbury** [comment]
I just tried putting my 4k display in 3840x1080. It doesn't work with borderless either.

It's like it's determining hitboxes using proportions of the full screen size but then only rendering in the centre. If you enter the game it also only draws in the centre of the screen with the standard aspect ratio although that might be necessary I suppose.

**Comment by thisismynombre** [comment]
I am having the same issue.  It works correctly when I untick fullscreen, but does not work for just unticking borderless.  

Additionally, when I take the game out of fullscreen, it will only hang out in the upper left, I can't move the window.  The cursor is perfectly accurate in windowed mode, but playing the game in the upper left of the screen isn't ideal.  I tried Alt+Space,M which would generally let you move a window with the arrow keys, but that didn't work either.

Interestingly, the inaccuracy of the cursor seems to be based on where it is in the window.  In the tutorial, I was able to click accurately on things in the center of the screen, but as they got further from the center, the cursor was further off the mark.

What it looks like with fullscreen unticked:  https://photos.app.goo.gl/K5UkiuAasNX...

Resolution:  2560x1080
OS:  Windows 10
